<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerDetails</class>
 <widget class="QDialog" name="PlayerDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Player details</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/icons/floco-128.png</normaloff>:/icons/icons/floco-128.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="photo">
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>128</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources.qrc">:/icons/icons/player-128.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="txtPlayerName">
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Player name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="txtNumber">
         <property name="text">
          <string>Number:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="txtPost">
         <property name="text">
          <string>Post:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="txtBirth">
         <property name="text">
          <string>Day of birth:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="txtAddress">
         <property name="text">
          <string>Address:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="txtPhone">
         <property name="text">
          <string>Phone:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="txtMail">
         <property name="text">
          <string>E-mail:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Statistics</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>88</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
